packages/components/src/modifiers/hds-clipboard.ts

```diff
@@ -129,15 +136,35 @@ export const copyToClipboard = async (text, target) => {
 // because it's quite simple in its logic, and doesn't require injecting services
 // see: https://github.com/ember-modifier/ember-modifier#function-based-modifiers
 
-export default modifier((element, positional, named) => {
+export default modifier((element, _positional, named) => {
```

Comment:
When porting the popover-related modifiers (https://github.com/hashicorp/design-system/pull/2103 and https://github.com/hashicorp/design-system/pull/2104) I've tried to follow a signature pattern that I've found on other codebases (eg. https://github.com/cardstack/cardstack/blob/dc2e0f9a42837feb073e3a3033b1a9188f50b4f7/packages/boxel/addon/modifiers/autoscroll.ts#L8-L14 or https://github.com/ember-bootstrap/ember-bootstrap/blob/6e9f791fd065a35e680cf26217682b7779ec19aa/ember-bootstrap/addon/modifiers/bs-conditional-attribute.ts#L3-L8 or https://github.com/adopted-ember-addons/ember-popper-modifier/blob/main/addon/-base-popper-modifier.ts#L76)

Maybe we can use a similar pattern here: see for example this https://github.com/hashicorp/design-system/blob/popover-primitive-ts-conversion/packages/components/src/modifiers/hds-register-event.ts

Reply:
I've asked in the channel to confirm this: https://hashicorp.slack.com/archives/C06NMSYDZPA/p1715956057393179